spec/models/season/phase_switcher_spec.rb

```diff
@@ -54,7 +54,7 @@
     it 'fails silently when it receives a non-whitelisted phase' do
       phase = 'bad_intentions'
       forbidden_call = Season::PhaseSwitcher.destined(phase)
-      expect { forbidden_call }.not_to change { Season.current.updated_at }
+      expect { forbidden_call }.not_to change { Season.current.updated_at.to_s }
     end
```

Comment:
I'm not entirely sure if that's really about rounding. The timestamp shouldn't change at all. Furthermore, the `forbidden_call` gets evaluated in line 56 whereas it has to be evaluated within the `expect { }` block. Can you update that one and we'll check the timestamp issue after that?